modules/ROOT/pages/_partials/endpoint-https-send.adoc

```diff
@@ -1,6 +1,6 @@
 == HTTP(S) Send Configuration
 
-Use these settings when sending with the secure HTTP(S) protocols.
+Use these settings when sending with HTTP and the secure HTTPS protocol.
 This includes endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
 
```

Comment:
Going forward, avoid using a table for this much information in only two columns. Anything longer than a line or two in a table cell is not considered scannable, which is the intent when using a table. GLOBAL.

Reply:
Got it. As I mentioned, I was not able to get a resolution from the PM in time for GA1. Hope to make a global change to the edit style for the next release.